docs/release-notes/det-pass.txt

```diff
@@ -0,0 +1,7 @@
+:orphan:
+
+**Improvements**
+
+-  CLI: Commands now support setting the environment variables ``DET_USER`` and ``DET_PASS`` to skip
```

Comment:
gosh, this is the tiniest nit, but... can you make this say "The ``det`` CLI command now supports" instead?

My reason is that "commands" refers to a special thing in our system, so upon first reading it, I skimmed a bit too fast and thought you were talking about `det cmd run` type of "commands".